### DISCUSSION TITLE
Sub Workflow

### DISCUSSION THREAD

**Discussion by malinru** [elsa-workflows/elsa-core#789] [Q&A]
It can trigger a workflow on another computer and wait for its execution result？how to do？

**Comment by sfmskywalker** [comment]
Elsa 2 can indeed, by the power invested in the `RunWorkflow` activity. When using this activity, you can specify the ID of the workflow to execute, as well as things like input, correlation ID and a `Mode` which specifies how to run this workflow:

- FireAndForget
- Blocking

The `FireAndForget` mode starts the sub workflow and continues with its own workflow.

The `Blocking` mode will also start the sub workflow, and wait for it to finish. If the sub workflow itself gets suspended due to a blocking activity, the parent workflow also gets suspended until the sub workflow ultimately gets resumed and finishes.
